tests/test_executor.py

```
@@ -119,6 +121,108 @@ def test_executor_init_bad_tmp_path(
+    tmpdir.mkdir(exist_ok=True)
+    datadir, logfile_path = process._prepare_dir(tmpdir, port)
+
+    executor = PostgreSQLExecutor(
+        executable=pg_exe,
+        host=config.host,
+        port=port,
+        datadir=str(datadir),
+        unixsocketdir=str(tmpdir / "socket dir"),
+        logfile=str(logfile_path),
+        startparams=config.startparams,
+        password="test pass",
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to argument: "password"

(S106)